persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java

```diff
@@ -191,7 +193,11 @@ protected void authorizeByToken(Map<String, ByteBuffer> customPayload, CQLStatem
 
       try {
         authorization.authorizeDataRead(
-            authenticationSubject, castStatement.keyspace(), castStatement.table(), SourceAPI.CQL);
+            authenticationSubject,
+            castStatement.keyspace(),
+            castStatement.table(),
+            SourceAPI.CQL,
+            EMPTY_HEADERS);
```

Comment:
Same concern as in https://github.com/stargate/stargate/pull/581/files#r559749974

Reply:
For stargate query handlers we should pass EMPTY_HEADERS. Only the CNDB StargasteQueryHandler will extract the tenant id from ClientStateWithTenant and pass it to the components. It was discussed on slack.